client/state/i18n/reducer.js

```diff
@@ -0,0 +1,11 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import localeSuggestions from './locale-suggestions/reducer';
+import { combineReducers } from 'state/utils';
+
+export default combineReducers( {
+	localeSuggestions,
+} );
```

Comment:
Looks like we didn't add i18n to the top level reducer?

Eg in console we should see:
```
dispatch({ type:'I18N_LOCALE_SUGGESTIONS_REQUEST' })
state.i18n.localeSuggestions.items //should be populated.
```

Reply:
This is good to 🚢  after this is updated.

Reply:
> Looks like we didn't add i18n to the top level reducer?
Argh. The most important thing and I missed it. 😟 Thanks!

Reply:
yep and I think it's perfectly valid to just let it be `state.i18n.localeSuggestions` as in my previous comment mentioned